target/linux/msm89xx/config-6.1

```diff
@@ -0,0 +1,6244 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Linux/arm64 6.1.0 Kernel Configuration
+#
+CONFIG_CC_VERSION_TEXT="aarch64-alpine-linux-musl-gcc (Alpine 12.2.1_git20220924-r6) 12.2.1 20220924"
```

Comment:
Please update the configuration like this:
````
make kernel_oldconfig CONFIG_TARGET=target
````

Reply:
Thanks, Updated it, then repeated the oldconfig using whatever .config I found in the build dir after it has succeeded since the build was still failing with some questions otherwise.

Reply:
If you still get some questions deactivate some options in the  `target/linux/generic/config-6.1` file.